.gitignore

```diff
@@ -136,3 +136,4 @@ reinforcement_learning/unit_test/rlclient-test.out
 cs/azure_service/azure_service.ccproj.user
 
 build
+dist/
```

Comment:
why is this needed? what is creating this dir?

Reply:
The directory `dist/` is generated when `python setup.py install` is run. 

It is clearly mentioned [here](https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives)

> command should output a lot of text and once completed should generate two files in the dist directory

We do not want this to be added to our git history.